backend/siarnaq/bracket/challonge.py

```diff
@@ -0,0 +1,269 @@
+# An API-esque module for our usage.
+# Commands here are not very generic (like a good API),
+# and are instead tailored to Battlecode's specific usage,
+# to improve dev efficiency
+
+from __future__ import annotations
+
+import json
+from typing import TYPE_CHECKING
+
+import requests
+from django.apps import apps
+from django.conf import settings
+
+if TYPE_CHECKING:
+    from typing import Iterable
+
+    from siarnaq.api.compete.models import Match
+    from siarnaq.api.episodes.models import Tournament, TournamentRound
+    from siarnaq.api.teams.models import Team
+
+
+_headers = {
+    "Accept": "application/json",
+    "Authorization-Type": "v1",
+    "Authorization": settings.CHALLONGE_API_KEY,
+    "Content-Type": "application/vnd.api+json",
+    # requests' default user agent causes Challonge's API to crash.
+    "User-Agent": "",
+}
+
+AUTH_TYPE = "v1"
+URL_BASE = "https://api.challonge.com/v2/"
+
+
+def create_tournament(
+    tournament: Tournament,
+    is_private,
+):
```

Comment:
You can annotate `is_private`. For readability and SFB, force it to be a kwarg. Do the same with all other methods in this module.
```suggestion
def create_tournament(
    tournament: Tournament,
    *
    is_private: bool,
):
```